package.json

```diff
@@ -5,8 +5,8 @@
   "main": "./src/client.js",
   "scripts": {
     "amp:validate": "wait-on http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/igbo.amp && amphtml-validator http://localhost.bbc.com:7080/korean/bbc_korean_radio/liveradio.amp",
-    "bbcA11y": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po && bbc-a11y",
-    "bbcA11y:ci": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po && xvfb-run -a bbc-a11y",
+    "bbcA11y": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/cn7k01xp8kxo && bbc-a11y",
+    "bbcA11y:ci": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/cn7k01xp8kxo && xvfb-run -a bbc-a11y",
```

Comment:
How does `bbc-a11y` know which asset to test?

Reply:
@jamesdonoh [it uses](https://github.com/bbc/simorgh/blob/latest/a11y.js#L4) the path set in [cypress services config](https://github.com/bbc/simorgh/blob/latest/cypress/support/config/services.js#L776)

Reply:
Ah okay, so the article ID we specify on these two lines is irrelevant then. But we might as well change it for consistency.